consensus/src/epoch_manager.rs

```diff
@@ -561,8 +561,8 @@ impl<P: OnChainConfigProvider> EpochManager<P> {
         max_blocks_allowed: u64,
     ) {
         let (request_tx, mut request_rx) = aptos_channel::new::<_, IncomingBlockRetrievalRequest>(
-            QueueStyle::LIFO,
-            1,
+            QueueStyle::KLAST,
```

Comment:
Why do you change these channels?

Reply:
We should ideally process the messages in FIFO order right?
With KLAST, oldest messages are dropped, but remaining are retrieved in FIFO order.